vspace/src/main/java/edu/asu/diging/vspace/core/factory/impl/AboutPageFormFactory.java

```diff
@@ -0,0 +1,47 @@
+package edu.asu.diging.vspace.core.factory.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import edu.asu.diging.vspace.core.factory.IAboutPageFormFactory;
+import edu.asu.diging.vspace.core.model.IExhibition;
+import edu.asu.diging.vspace.core.model.IExhibitionLanguage;
+import edu.asu.diging.vspace.core.model.impl.ExhibitionAboutPage;
+import edu.asu.diging.vspace.core.services.IExhibitionManager;
+import edu.asu.diging.vspace.web.staff.forms.AboutPageForm;
+
+@Component
+public class AboutPageFormFactory  implements IAboutPageFormFactory{
```

Comment:
the form related classes should be in the `web` package since that's where they are being used.